package.json

```diff
@@ -10,6 +10,7 @@
     "build:init": "./bin/update-version.sh && mkdir -p dist && cp -r public/* dist/ && npm run db:build",
     "processes:build": "elm make src/ComputeAggregated.elm --output=compute-aggregated-app.js && node compute-aggregated.js",
     "db:build": "./bin/build-db",
+    "db:build:full": "TEXTILE_PROCESSES=public/data/textile/processes_impacts.json FOOD_PROCESSES=public/data/food/processes_impacts.json ./bin/build-db",
```

Comment:
Wondering out loud, couldn't we get rid of the `TEXTILE_PROCESSES` and `FOOD_PROCESSES ` env vars entirely? And/or we could infer the file paths by checking the `DJANGO_BYPASS_AUTH` value (and if so maybe find a better name for it?)